kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/util/KuranetConfig.java

```diff
@@ -128,7 +127,9 @@ public static void storeProperties(Properties props) throws IOException, KuraExc
                     s_logger.info("Not rewriting kuranet props file because it is the same");
                 }
             } finally {
-                fos.close();
+            	if (fos != null) {
+            		fos.close();
```

Comment:
It looks like you are not using the [formatters](https://github.com/eclipse/kura/blob/release-3.0.0/CONTRIBUTING.md)